testflinger_agent/client.py

```diff
@@ -251,3 +251,17 @@ def post_images(self, data):
             requests.post(images_uri, json=data, timeout=30)
         except Exception as e:
             logger.exception(e)
+
+    def post_agent_data(self, data):
+        """Post the relevant data points to testflinger server
+
+        :param data:
+            dict of various agent data points to send to the api server
+        """
+        agent = self.config.get("agent_id")
+        agent_data_uri = urljoin(self.server, "/v1/agents/data")
+        agent_data_url = urljoin(agent_data_uri, agent)
+        try:
+            self.session.post(agent_data_url, json=data, timeout=30)
+        except Exception as e:
+            logger.exception(e)
```

Comment:
using the same session to minimize new requests connections. it appears that agent 'state' changes fairly frequently, so I saw this as advantageous.

Reply:
ok, but please don't add more "except Exception" handlers. I know we already have this in some other places, but we don't want to add more that we'll just have to clean up later.